pages/blog/houdini-and-kitql.mdx

```diff
@@ -0,0 +1,216 @@
+import withArticle from '../../ui/blog/article'
+
+export const meta = {
+  title: 'KitQL & Houdini projects are joining forces',
```

Comment:
I would add something to the title and description for people who don't understand what these libraries are,
something exciting with the words Sveltle and GraphQL in them.....

Reply:
Title: "Strive to bring the best GraphQL experience in Svelte"
Description: "It's time to unveil a massive effort by the KitQL and Houdini maintainers to join forces!"

Other ideas?

Reply:
We are happy about: "Bringing the best GraphQL experience to Svelte"